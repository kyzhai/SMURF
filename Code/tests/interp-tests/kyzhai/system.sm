a = [[(1,2)$1.., (2,3)$1..], [(3,3)$1.., (10,3)$1..]]

main = a
