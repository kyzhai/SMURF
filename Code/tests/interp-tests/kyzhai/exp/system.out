
Timing Resolution set to 4 PPQ

Instrument set to 48 on channel 0

Track 0:
 tick 0, channel 1: program change 48
 tick 0, channel 1: note C#4 on velocity: 90
 tick 0, channel 1: note D5 on velocity: 90
 tick 28, channel 1: note C#4 on velocity: 0
 tick 28, channel 1: note D5 on velocity: 0
 tick 28, channel 1: note D#5 on velocity: 90
 tick 28, channel 1: note A#5 on velocity: 90
 tick 56, channel 1: note D#5 on velocity: 0
 tick 56, channel 1: note A#5 on velocity: 0
 tick 56, end of track
PASSED SEMANTIC CHECKS
java -jar ./Lib/CSV2MIDI.jar a.csv a.midi
===== Program Successfully Finished =====
===== Result Writen to a.midi =====
